lib/ssl/src/tls_handshake_1_3.erl

```diff
@@ -1302,7 +1306,18 @@ process_certificate(#certificate_1_3{certificate_list = Certs0},
                                             SslOptions, CRLDbHandle, Role, Host) of
                 {ok, {PeerCert, PublicKeyInfo}} ->
                     State = store_peer_cert(State0, PeerCert, PublicKeyInfo),
-                    {ok, {State, wait_cv}};
+                    NewState = check_ocsp(State, Certs0),
```

Comment:
We should refactor this part. process_certificate should use the negotiated OCSP state set when we received the ServerHello. All this should be handled in the validate_certificate_chain function.